src/Streamly/Prelude.hs

```diff
@@ -477,6 +498,12 @@ module Streamly.Prelude
     -- -------Stream m a------|
     -- @
     --
+    , serial
+    , wSerial
+    , ahead
+    , async
+    , wAsync
+    , parallel
```

Comment:
`serial` and `ahead` should go in the appending section above. `wSerial` should go in a separate `Interleaving` section - see `IsStream` module, organize in the same way.